qpc/insights/tests_insights_publish.py

```diff
@@ -148,7 +196,7 @@ def test_insights_publish_successful(  # pylint: disable=too-many-arguments
             "insights",
             "publish",
             "--input-file",
-            payload_file.name,
+            str(payload_file),
```

Comment:
why not use name attribute ?

Reply:
it wont give the full path for the file. I didn't dig on the explanation, but the file produced by `tmp_path` fixture don't behave the same way as `NamedTemporaryFile`

Reply:
Now that I thought "out loud" about it makes sense to behave differently. `tmp_path` is a `pathlib.Path` object in the end :thinking:

Reply:
got it.

Reply:
https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.name